config/locales/views/submissions/en.yml

```diff
@@ -79,5 +79,6 @@ en:
       only_required_files: You may only submit the required files listed above.
       select_file: Select a file to view
       version_control_warning: Submitting files here overrides any changes made using version control. Make sure you have worked on the most recent files before submitting.
+    submission_summary: Submission Summary
```

Comment:
Let's change this to "Submission Report" (I'm cautious about using the word "Summary" too often)